webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/node/AcceptNode.scala

```diff
@@ -138,9 +138,9 @@ class AcceptNode extends Loggable {
       accept match {
         case f:     Failure  =>
           S.error(
-            <span>
-            {f.messageChain}
-          </span>
+            <div class="callout-fade  callout-danger">
+              <strong>{f.messageChain}</strong>
+            </div>
```

Comment:
I'd prefer to use the .alert class rather than .callout :
```
<div class="alert alert-danger">
  <i class ="fa fa-warning"></i>
  <strong>{f.messageChain}</strong>
</div>
```

 It's not very clear which class to use in this kind of situation (we sometimes use .callout, sometimes .alert), it lacks consistency. There's no good reason for this, we should make a final decision on this.

Until then, the .alert style would allow this message to be even more noticeable than the information messages that precede it, and we tend to use this style of message in our latest interfaces
![alert-danger-preview](https://github.com/user-attachments/assets/92f705bd-573f-41cf-915c-42bdc8c012fa)